src/components/shared/Reorderable.js

```diff
@@ -77,43 +75,37 @@ class Reorderable extends React.PureComponent<Props, State> {
     };
   }
 
-  _setContainerRef(container: HTMLElement | null) {
-    this._container = container;
-  }
+  _onMouseDown(
+    event: { target: EventTarget } & SyntheticMouseEvent<HTMLElement>
+  ) {
+    const container = event.currentTarget;
 
-  _onMouseDown(event: SyntheticMouseEvent<>) {
     if (
-      !this._container ||
-      event.currentTarget === this._container ||
-      !(event.currentTarget instanceof HTMLElement) ||
+      event.target === container ||
+      !(event.target instanceof HTMLElement) ||
```

Comment:
I reintroduced the use of `target` here, because we use event delegation.
Yet using `currentTarget` actually eliminates the need for the `container` ref so that's what I did.